# content of b/test_error.py
def test_root(db):  # no db here, will error out
    pass